setup.py

```diff
@@ -0,0 +1,54 @@
+from setuptools import setup, find_packages
+from distutils.util import convert_path
+
+
+def _read_long_description():
+    with open("README.md") as readme:
+        return readme.read()
+
+
+DEV_REQUIRE = [
+    "pytest",
+    "pytest-cov",
+    "black",
+    "mypy",
+    "python-semantic-release",
+]
+NAME = "flaky_tests_detection"
+NAME_DASHED = NAME.replace("_", "-")
+OLD_GIT_URL = "fliprate_actions"  # Are we going to change repo name?
```

Comment:
Yes, let's rename it to `flaky_tests_detection`. You are granted admin privileges, but please before renaming sync with @tatu-aalto because `fliprate_actions` is already in use in one of the open-source projects.

Reply:
I am happy with name change. It is one liner to change and easy to spot when it breaks. And in any case, I want to run it in windows too, so most likely I will end up converting the actions as Python dependency.